.github/issue-close-app.yml

```diff
@@ -1,7 +1,17 @@
 # Config for Close Issue app: https://probot.github.io/apps/close-issue/
 
 # Comment that will be sent if an issue is judged to be closed
-comment: "This issue was closed because it does not use any of our issue templates. Please make sure to use one of the suggested templates."
+comment: |
+  This issue was closed because it does not use our bug report issue template.
+
+  Please make sure to use it and fill it as much as you can so we can provide better and fast support.
+
+  The issue **must** contain at least the following info:
```

Comment:
All contributors facing this have deleted sections that don't apply.   Suggest ...

The following sections **must** not be removed, or else the BOT will close it immediately again.

Reply:
"Version of Rocket.Chat Server" in the list is not actually a section, but I change the item to be "Server Setup Information" instead, wdyt?

Reply:
Sure, please do.    We need that entire section anyways!  Thanks.

Reply:
changed the text as you suggested =)

Reply:
LGTM